test/integration/retryable-reads/retryable_reads.spec.test.js

```diff
@@ -23,7 +23,7 @@ describe('Retryable Reads', function () {
       spec.description.match(/listCollections/i) ||
       spec.description.match(/listCollectionNames/i) ||
       spec.description.match(/estimatedDocumentCount/i) ||
-      // FIXME(NODE-3074): uncomment when `count` spec tests have been fixed
+      // FIXME(NODE-3369): uncomment when `count` spec tests have been fixed
       // spec.description.match(/count/i) ||
```

Comment:
Do these run successfully now?, or is this an inclusion list?

Reply:
inclusion

Reply:
I was _just_ looking at this exact line on my other PR and wondering why NODE-3074 was linked here 😄